src/wwwroot/controllers/TemplateCtrl.js

```diff
@@ -57,7 +57,7 @@
       }
 
       $scope.saveInProgress = true;
-      templates.save({
+      return templates.save({
```

Comment:
❤️ 
Although it was not necessary I was expressing my opinion, not asking you to apply a change right now. but thanks.

Reply:
I like your opinion about why is good idea implement it 👍